### PR TITLE
Derek/7051 groups billing update

### PR DESCRIPTION
**Pull request by dangle**
Change for #7051 to have calculate_price use the billing info API instead of hard coded values.

### CODE REVIEW

**Comment by Bennyoak** [comment]
Looks good to me.

File: django/apps/blue_management/blue_mgnt/static/js/billing.js [hunk 1]
```diff
@@ -58,14 +58,26 @@
...
+    var yearly_cost = 60;
+    $.ajax("/billing/info", {
+        type: "GET",
+        // FIXME: "async: false" deprecated but simplest solution for now
+        async: false,
+        success: function(data, status, xhr) {
+                monthly_cost = "monthly_cost" in data ? data["monthly_cost"] : 5;
+                yearly_cost = "yearly_cost" in data ? data["yearly_cost"] : 60;
+            },
+    });
```
Comment (bdzim) [inline django/apps/blue_management/blue_mgnt/static/js/billing.js]: What happens if this ajax fails? Could the page potentially show the user the wrong price?

Comment (dangle) [inline django/apps/blue_management/blue_mgnt/static/js/billing.js]: If it fails, it will show the current hard coded values.

Comment (bdzim) [inline django/apps/blue_management/blue_mgnt/static/js/billing.js]: Can we move the billing info call to the view so there's no possibility of showing the wrong values?

**Comment by qdonnellan** [comment]
Seems good to me

**Comment by bdzim** [comment]
Looks good. We can merge after it passes testing.

**Pull request merged**